homeassistant/components/sfr_box/sensor.py

```diff
@@ -120,14 +120,15 @@ class SFRBoxSensorEntityDescription(SensorEntityDescription, SFRBoxSensorMixin):
         entity_category=EntityCategory.DIAGNOSTIC,
         entity_registry_enabled_default=False,
         options=[
-            "No Defect",
-            "Of Frame",
-            "Loss Of Signal",
-            "Loss Of Power",
-            "Loss Of Signal Quality",
-            "Unknown",
+            "no-defect",
+            "of-frame",
+            "loss-of-signal",
+            "loss-of-power",
+            "loss-of-signal-quality",
+            "unknown",
         ],
-        value_fn=lambda x: x.line_status,
+        translation_key="line_status",
+        value_fn=lambda x: x.line_status.lower().replace(" ", "-"),
```

Comment:
While the hyphen is accepted, the use of it is very very rare, so, maybe just go with the most common occurrence and use an underscore instead>?

```suggestion
        value_fn=lambda x: x.line_status.lower().replace(" ", "_"),
```